torch/csrc/xpu/Module.cpp

```diff
@@ -274,6 +274,26 @@ PyObject* THXPModule_resetAccumulatedMemoryStats(
   Py_RETURN_NONE;
 }
 
+PyObject* THXPModule_getMemoryInfo(PyObject* self, PyObject* arg) {
```

Comment:
nit: use pybind11 for this kind of functions. It makes the code simpler to maintain.

Reply:
Thanks, @albanD, pybind11 makes my life easier. We will file another PR to bind these methods left in this file via pybind11.